src/sentry/api/serializers/models/project.py

```diff
@@ -183,17 +179,30 @@ def measure_span(op_tag):
         return result
 
     @staticmethod
-    def _get_feature_list(obj, user, project_features):
-        feature_list = set(
-            feature_name[len(_PROJECT_SCOPE_PREFIX) :]  # Remove the project scope prefix
-            for feature_name in project_features
-            if features.has(feature_name, obj, actor=user)
-        )
+    def _get_feature_lists(all_projects, user):
+        # Arrange to call features.has_for_organization rather than
+        # features.has for performance's sake
+        projects_by_org = defaultdict(list)
+        for project in all_projects:
+            projects_by_org[project.organization].append(project)
+
+        features_by_project = defaultdict(list)
+        for feature_name in features.all(feature_type=ProjectFeature).keys():
+            if feature_name.startswith(_PROJECT_SCOPE_PREFIX):
```

Comment:
codenit: can you turn this into a guard statment?

```py
if not feature_name.startswith(_PROJECT_SCOPE_PREFIX):
	continue
```

Reply:
Sure thing.